contracts/upgradeable_contracts/erc20_to_erc20/BasicForeignBridgeErcToErc.sol

```diff
@@ -34,7 +34,13 @@ contract BasicForeignBridgeErcToErc is BasicForeignBridge {
         return 0xba4690f5; // bytes4(keccak256(abi.encodePacked("erc-to-erc-core")))
     }
 
+    /**
+     * @dev Withdraws the erc20 tokens or native coins from this contract. Bridged token cannot be withdrawn by this function.
+     * @param _token address of the claimed token or address(0) for native coins.
+     * @param _to address of the tokens/coins receiver.
+     */
     function claimTokens(address _token, address _to) public {
```

Comment:
Shouldn't it be external?

Reply:
Since this overrides and calls `super.claimTokens`, they both need to be public, this is forced by the solidity compiler.
But since we can get rid of `claimTokens` in `BasicBridge.sol`, it is possible to make it external.